server/services/teamsSummaryCountService.ts

```diff
@@ -151,4 +155,69 @@ export default class TeamsSummaryCountService {
       return {}
     }
   }
+
+  /**
+   * Helper: Get Trivy CRITICAL & HIGH vuln counts for a team's products
+   */
+  async getTeamTrivyVulnerabilityCounts(
+    products: StrapiProduct[],
+    serviceCatalogueService: ServiceCatalogueService,
+  ): Promise<{ critical: number; high: number }> {
+    try {
+      if (!Array.isArray(products) || products.length === 0) {
+        return { critical: 0, high: 0 }
+      }
+
+      const [trivyScans, allComponents] = await Promise.all([
```

Comment:
Don't need to do this, just do two `await` statements, it's easier to read than mixing `await/async` and promise stuff